config/checkerframework/CallableStatement.astub

```diff
@@ -0,0 +1,91 @@
+package java.sql;
+
+import org.checkerframework.checker.nullness.qual.*;
+
+interface CallableStatement {
+  @Nullable String getString(int parameterIndex);
+  @Nullable BigDecimal getBigDecimal(int parameterIndex, int scale);
+  byte @Nullable [] getBytes(int parameterIndex);
+  java.sql.@Nullable Date getDate(int parameterIndex);
+  java.sql.@Nullable Time getTime(int parameterIndex);
+  java.sql.@Nullable Timestamp getTimestamp(int parameterIndex);
+  @Nullable Object getObject(int parameterIndex);
+  @Nullable BigDecimal getBigDecimal(int parameterIndex);
+  @Nullable Object getObject(int parameterIndex, java.util.Map<String,Class<?>> map);
```

Comment:
can't the `Map` parameter be null?

Reply:
It can. Good catch.

The current verifier has the following checks:
1) It ensures that method has `@Nullable` in case the returned value can be `null`. For instance, if method contains `return null`, then the verifier ensures the return type is nullable
2) If a method is called with nullable argument, then the verifier ensures the parameter has nullable type.

For now, there are lots of violations left, and I guess after the initial pass is done, we would need to scan all the API methods and put extra `@Nullable` annotations to the methods which do accept `null`.